android/app/src/main/java/com/reactnativeapp/MainApplication.java

```diff
@@ -31,21 +32,22 @@ public boolean getUseDeveloperSupport() {
         }
 
         @Override
-        protected List<ReactPackage> getPackages() {
-            return Arrays.asList(
-                    new MainReactPackage(),
-                    new RNCWebViewPackage(),
-                    new NetInfoPackage(),
-                    new AsyncStoragePackage(),
-                    new MapsPackage(),
-                    new RNLoggingPackage(),
-                    new RNTranslationManagerPackage(),
-                    new RNCurrencyManagerPackage(),
-                    new RNDeviceInfoPackage(),
-                    new RNCardManagerPackage(),
-                    new RNGestureHandlerPackage()
-            );
-        }
+    protected List<ReactPackage> getPackages() {
+        @SuppressWarnings("UnnecessaryLocalVariable")
+        List<ReactPackage> packages = new PackageList(this).getPackages();
+        // Packages that cannot be autolinked yet can be added manually here, for example:
+        packages.add(new RNCWebViewPackage());
```

Comment:
Did you try to remove some of them here? I would think that most of them can be autolinked.

Reply:
Except our custom ones, of course.

Reply:
I tried, and it did not work. Feel free to give it a try. Also, we cannot autolink for Android in rnkiwimobile framework, since we fetch dependencies from maven, and not node modules